plugins/splunk-on-call/src/components/EntitySplunkOnCallCard.tsx

```diff
@@ -45,20 +45,50 @@ import {
 } from '@backstage/core-components';
 
 export const SPLUNK_ON_CALL_TEAM = 'splunk.com/on-call-team';
+export const SPLUNK_ON_CALL_ROUTING_KEY = 'splunk.com/on-call-routing-key';
 
-export const MissingTeamAnnotation = () => (
-  <MissingAnnotationEmptyState annotation={SPLUNK_ON_CALL_TEAM} />
+export const MissingAnnotation = () => (
+  <div>
+    <Typography>
+      The Splunk On Call plugin requires setting either the{' '}
+      <code>{SPLUNK_ON_CALL_TEAM}</code> or the{' '}
+      <code>{SPLUNK_ON_CALL_ROUTING_KEY}</code> annotation.
+    </Typography>
+    <MissingAnnotationEmptyState annotation={SPLUNK_ON_CALL_TEAM} />
+    <MissingAnnotationEmptyState annotation={SPLUNK_ON_CALL_ROUTING_KEY} />
```

Comment:
☝️ Note that we now display 2 `<MissingAnnotationEmptyState/>` components -- one for each of the supported annotations -- preceded by a sentence explaining that one of the annotations must be used. I struggled to imagine a better way to present this UX using current Backstage patterns, but perhaps others have ideas?

Reply:
I'd say the text above is helps out and that it's enough with one example for the most common annotation.

Reply:
@jhaals Thanks for the feedback -- I think that's a good suggestion and I've pushed a commit removing the second `<MissingAnnotationEmptyState />`.